### PR TITLE
Update RELEASE.md

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatoliy Bazko <abazko@redhat.com>

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -68,7 +70,6 @@ xdg-open http://$(kubectl get ingress -n che | grep ^che | awk -F ' ' '{ print $
 ## 5. Prepare community operator PR
 
 ```bash
-olm/prepare-community-operators-update.sh
+cd olm
+./prepare-community-operators-update.sh
 ```
-
-TODO automate creating PRs
```
Comment (nickboldt) [inline RELEASE.md]: PRs are now auto-generated?

Comment (tolusha) [reply RELEASE.md]: not yet

Comment (nickboldt) [reply RELEASE.md]: then why remove the reminder ?

Comment (tolusha) [reply RELEASE.md]: Restored

**Pull request merged**